.github/workflows/tests-integration.yml

```diff
@@ -54,44 +54,37 @@ jobs:
         optuna --version
 
         pip install --progress-bar off .[tests]
-        pip install --progress-bar off .[integration] -f https://download.pytorch.org/whl/torch_stable.html
-
-        # TODO(nzw0301): remove the version constraint
-        # when resolving https://github.com/optuna/optuna/issues/3418
-        pip install --progress-bar off -U "setuptools==59.5.0"
+        pip install --progress-bar off .[integration] --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
[note] `-f https://download.pytorch.org/whl/torch_stable.html` is still used in:  (🔗 https://pytorch.org/get-started/previous-versions) but `--extra-index-url https://download.pytorch.org/whl/cpu` is suggested in (🔗 https://deploy-preview-972--pytorch-dot-org-preview.netlify.app/get-started/locally/).

(🔗 https://github.com/pytorch/pytorch.github.io/pull/972)